structuralcodes/core/_section_results.py

```diff
@@ -175,37 +175,86 @@ class UltimateBendingMomentResults:
 
 
 @dataclass
-class NMMInteractionDomain:
+class InteractionDomain:
+    """Class for storing common data on all interaction domain results.
+
+    Data contained:
+
+    strains: a numpy array with shape (n, 3) containing ea, ky and kz.
+    forces: a numpy array with shape (n, 3) containing n, my and mz.
+    field_num: a numpy array with shape (n,) cotaning a number between
```

Comment:
Minor typo: cotaining -> containing.

Reply:
done in 6b1ed715f89c7ad4f5f0772b8d48e73501484073